vhdbuilder/packer/install-dependencies.sh

```diff
@@ -668,6 +668,60 @@ else
 fi
 capture_benchmark "${SCRIPT_NAME}_finish_installing_bcc_tools"
 
+# Configure LSM modules to include BPF
+configureLsmWithBpf() {
+  echo "Configuring LSM modules to include BPF..."
+
+  # Read current LSM modules
+  if [ ! -f /sys/kernel/security/lsm ]; then
+    echo "Warning: /sys/kernel/security/lsm not found, skipping LSM configuration"
+    return 0
+  fi
+
+  local current_lsm=$(cat /sys/kernel/security/lsm)
+  echo "Current LSM modules: $current_lsm"
+
+  # Prepend bpf to the LSM list if not already present
+  if [[ "$current_lsm" != *"bpf"* ]]; then
```

Comment:
shouldnt this done only when Cilium is used ?

Reply:
As discussed offline, we can't check for Cilium as part of node image creation. So enabling on 2404 and azl 3.0.